From 4b152645bab71f3d36c4c3169a7c49d7948e0de8 Mon Sep 17 00:00:00 2001
From: San Mehat <san@google.com>
Date: Wed, 16 Sep 2009 12:39:10 -0700
Subject: [PATCH] proc: smaps: Allow smaps access for CAP_SYS_RESOURCE

Signed-off-by: San Mehat <san@google.com>
---
 kernel/fork.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/kernel/fork.c b/kernel/fork.c
index a17621c..dfb7139 100644
--- a/kernel/fork.c
+++ b/kernel/fork.c
@@ -690,7 +690,8 @@ struct mm_struct *mm_access(struct task_struct *task, unsigned int mode)
 
 	mm = get_task_mm(task);
 	if (mm && mm != current->mm &&
-			!ptrace_may_access(task, mode)) {
+			!ptrace_may_access(task, mode) &&
+			!capable(CAP_SYS_RESOURCE)) {
 		mmput(mm);
 		mm = ERR_PTR(-EACCES);
 	}
-- 
1.8.0

