From b17599de9c62257fbd85f4fc250f5b848ba8d5eb Mon Sep 17 00:00:00 2001
From: Brian Swetland <swetland@google.com>
Date: Thu, 21 Oct 2010 14:19:31 -0700
Subject: [PATCH] staging: remove Greg's TODO, now obsolete.

Signed-off-by: Brian Swetland <swetland@google.com>
---
 drivers/staging/android/TODO | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/staging/android/TODO b/drivers/staging/android/TODO
deleted file mode 100644
index b15fb0d..0000000
--- a/drivers/staging/android/TODO
+++ /dev/null
@@ -1,10 +0,0 @@
-TODO:
-	- checkpatch.pl cleanups
-	- sparse fixes
-	- rename files to be not so "generic"
-	- make sure things build as modules properly
-	- add proper arch dependencies as needed
-	- audit userspace interfaces to make sure they are sane
-
-Please send patches to Greg Kroah-Hartman <greg@kroah.com> and Cc:
-Brian Swetland <swetland@google.com>
-- 
1.8.0

