From a436cc575e6a66b4f260968c3a76bf49197f0970 Mon Sep 17 00:00:00 2001
From: Fathi Boudra <fathi.boudra@linaro.org>
Date: Mon, 27 May 2013 10:49:08 +0300
Subject: [PATCH] ubuntu-minimal.conf: add CONFIG_IP[6]_NF_FILTER=m

Signed-off-by: Fathi Boudra <fathi.boudra@linaro.org>
---
 linaro/configs/ubuntu-minimal.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/linaro/configs/ubuntu-minimal.conf b/linaro/configs/ubuntu-minimal.conf
index e3c0c0c..330adcf 100644
--- a/linaro/configs/ubuntu-minimal.conf
+++ b/linaro/configs/ubuntu-minimal.conf
@@ -8,7 +8,9 @@ CONFIG_SYN_COOKIES=y
 CONFIG_IPV6=y
 CONFIG_NETLABEL=y
 CONFIG_IP_NF_IPTABLES=m
+CONFIG_IP_NF_FILTER=m
 CONFIG_IP6_NF_IPTABLES=m
+CONFIG_IP6_NF_FILTER=m
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
 CONFIG_BLK_DEV_RAM=y
-- 
1.8.0

