From e82e74f723383f6f897f257089029a9bb65c8e32 Mon Sep 17 00:00:00 2001
From: Riku Voipio <riku.voipio@linaro.org>
Date: Wed, 20 Feb 2013 14:48:11 +0200
Subject: [PATCH] kvm-host: set CONFIG_VIRTUALIZATION=y

---
 linaro/configs/kvm-host.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linaro/configs/kvm-host.conf b/linaro/configs/kvm-host.conf
index 2d79d94..f47a2b0 100644
--- a/linaro/configs/kvm-host.conf
+++ b/linaro/configs/kvm-host.conf
@@ -1,3 +1,4 @@
+CONFIG_VIRTUALIZATION=y
 CONFIG_ARM_LPAE=y
 CONFIG_ARM_VIRT_EXT=y
 CONFIG_HAVE_KVM_IRQCHIP=y
-- 
1.8.0

