From 356838905e621a965808c26086861f46d7b2651f Mon Sep 17 00:00:00 2001
From: Robin Murphy <robin.murphy@arm.com>
Date: Mon, 4 Aug 2014 16:42:51 +0100
Subject: [PATCH] usb: Fix H20AHB driver for big-endian

Signed-off-by: Robin Murphy <robin.murphy@arm.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
---
 drivers/usb/host/ehci-h20ahb.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/host/ehci-h20ahb.c b/drivers/usb/host/ehci-h20ahb.c
index 3ee3c7a..7724bab 100644
--- a/drivers/usb/host/ehci-h20ahb.c
+++ b/drivers/usb/host/ehci-h20ahb.c
@@ -58,12 +58,12 @@ struct h20ahb_hcd {
 
 static inline void ehci_write(void __iomem *base, u32 reg, u32 val)
 {
-	__raw_writel(val, base + reg);
+	writel_relaxed(val, base + reg);
 }
 
 static inline u32 ehci_read(void __iomem *base, u32 reg)
 {
-	return __raw_readl(base + reg);
+	return readl_relaxed(base + reg);
 }
 
 /* configure so an HC device and id are always provided */
-- 
1.8.0

